ibtk/include/ibtk/IBTKInit.h

```diff
@@ -103,6 +106,7 @@ class IBTKInit
 #ifdef IBTK_HAVE_LIBMESH
     static libMesh::LibMeshInit* s_libmesh_init;
 #endif
+    static bool s_initialized;
```

Comment:
Can you do initialization in the header?

Reply:
I think you can only initialize `static const` member variables in headers.